pandas/io/hdfs.py

```diff
@@ -0,0 +1,27 @@
+""" s3 support for remote file interactivity """
+from pandas import compat
+try:
+    import hdfs3
+except:
+    raise ImportError("The hdfs3 library is required to handle hdfs files")
+
+if compat.PY3:
```

Comment:
these should be moved to ``pandas/compat/__init__.py`` (and same change in pandas.io.s3fs)